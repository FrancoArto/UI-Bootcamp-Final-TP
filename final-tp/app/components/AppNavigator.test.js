import React from 'react';

import ShallowRenderer from 'react-test-renderer/shallow';
import { AppNavigator } from './AppNavigator';


it('renders without crashing', () => {
  const rendered = new ShallowRenderer()
  rendered.render(<AppNavigator />)
  expect(rendered).toMatchSnapshot();
});