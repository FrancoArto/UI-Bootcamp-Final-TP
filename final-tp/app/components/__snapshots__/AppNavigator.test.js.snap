// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ReactShallowRenderer {
  "_context": Object {},
  "_element": <AppNavigator />,
  "_forcedUpdate": false,
  "_instance": AppNavigator {
    "context": Object {},
    "props": Object {},
    "refs": Object {},
    "state": null,
    "updater": Updater {
      "_callbacks": Array [],
      "_renderer": [Circular],
    },
  },
  "_newState": null,
  "_rendered": <NavigationContainer />,
  "_rendering": false,
  "_updater": Updater {
    "_callbacks": Array [],
    "_renderer": [Circular],
  },
}
`;
